.travis.yml

```diff
@@ -9,11 +9,11 @@ matrix:
         - lint
       script:
         - 
-          go get github.com/alecthomas/gometalinter && gometalinter --install && gometalinter ./... --deadline=10m --exclude=client/internal/client_helper.go
+          npm install -g solium && go get github.com/alecthomas/gometalinter && gometalinter --install && gometalinter ./... --deadline=10m --exclude=client/internal/client_helper.go
```

Comment:
Revert this. You're installing solium in the linter container and it won't be available to to the testing container.